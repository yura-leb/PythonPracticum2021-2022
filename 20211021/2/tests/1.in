:f 46
f
f
f
f
quit
