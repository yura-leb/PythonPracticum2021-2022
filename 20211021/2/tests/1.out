46
46
46
46
2:6
